merkle/common_test.go

```diff
@@ -74,19 +73,19 @@ func setupProofTestCases() ([]ProofTestCases, error) {
 	return merkleProofCases, nil
 }
 
-func combinations(leaves []types.Leaf) [][]types.Leaf {
-	return combine([]types.Leaf{}, leaves, [][]types.Leaf{})
+func combinations(leaves Leaves) Layers {
+	return combine(Leaves{}, leaves, Layers{})
 }
 
-func combine(active []types.Leaf, rest []types.Leaf, combinations [][]types.Leaf) [][]types.Leaf {
+func combine(active Leaves, rest Leaves, combinations Layers) Layers {
```

Comment:
can you please document what does `combine` do?

Reply:
fixed in [this](https://github.com/ComposableFi/go-merkle-trees/pull/22/commits/2e360cdbbfc6c3e12c3c206b0c3b3d92e91c702c)